docs/CRLS.md

```diff
@@ -0,0 +1,74 @@
+# CRLs
+
+For each issuer certificate, Boulder generates several sharded CRLs.
+The responsibility is shared across these components:
+
+ - crl-updater
+ - ca
+ - crl-storer
+ - sa
```

Comment:
These are neither in alphabetical order, nor the order that they're touched during the process (updater, sa, ca, storer). Is there a logic behind this ordering?

Reply:
No logic; I updated to go with "order they are touched".